### PR TITLE
doc/*: add man pages

### PR DESCRIPTION
**Pull request by fluorescent-haze**
- doc/tinyramfs.8: add manpage for tinyramfs
- doc/tinyramfs.5: add configuration manpage
- Makefile: install manpages

### CODE REVIEW

**Review by illiliti — CHANGES_REQUESTED** [review]
Thank you. Looks pretty good overall. I can merge this, but i would prefer if this would be rewritten in scdoc format. What do you think?

**Comment by fluorescent-haze** [comment]
Thanks for taking a look. It's my first time writing a man page, so I'm bound to make mistakes. I'll fix the things you pointed out. I wrote it in mdoc, but I'll take a look at scdoc.

**Comment by fluorescent-haze** [comment]
I rewrote the man pages in scdoc with your requested changes, which has a very nice syntax compared to plain mdoc. I also added a job in the Makefile to 'make doc' which just runs the doc/*.scd files through scdoc. Lemme know if there's anything else you want me to change.

File: doc/tinyramfs.5.scd [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Option marked with *(bool)* can hold a value of true or false. False if the 
+variable is unset or empty, true otherwise.
+
+Some options only apply to certain hooks and have been categorized accordingly.
+
+Options which expect a device as a value can also refer to the device via 
+*UUID*, *LABEL*, and *PARTUUID* (eg. LABEL=<label>). This requires a blkid 
+utility which supports that identifier.
+
+root
```
Comment (illiliti) [inline doc/tinyramfs.5.scd:21]: Add additional newline between option and description here. Do same for other options.

File: doc/tinyramfs.5.scd [hunk 2]
```diff
@@ -0,0 +1,98 @@
+tinyramfs(5) "tinyramfs" "2022-05-09"
+# NAME
```
Comment (illiliti) [inline doc/tinyramfs.5.scd:2]: Add additional newline between title and text here. Do same for others.

**Review by illiliti — CHANGES_REQUESTED** [review]
Great work! Here's some nitpicks which needs to fixed before i can merge this. Thanks again.

File: doc/tinyramfs.5.scd (hunk 1 above)
Comment (fluorescent-haze) [reply doc/tinyramfs.5.scd:21]: Multiple newlines don't show up in the output file, so I'll use '++' so it does, as I assume you'd want.

**Comment by fluorescent-haze** [comment]
Everything should be good now!

**Pull request closed without merging**